Sources/joyfill-components-swift/Components/Table/CollectionView/CollectionViewCell.swift

```diff
@@ -102,65 +93,19 @@ class CollectionViewCell: UICollectionViewCell, UITextFieldDelegate, UITextViewD
     
     // Function to add textView in contentView
     func setupTextView() {
-        contentView.addSubview(textView)
-        textView.translatesAutoresizingMaskIntoConstraints = false
-        
-        NSLayoutConstraint.activate([
-            textView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 0),
-            textView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
-            textView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor,constant: -1),
-            textView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -1)
-        ])
-        textView.layer.borderWidth = 0
-        textView.showsVerticalScrollIndicator = false
-        textView.font = UIFont.systemFont(ofSize: 12)
-        textView.textContainerInset = UIEdgeInsets(top: 12, left: 22, bottom: 5, right: 20)
-    }
-    
-    // Function to add textField in contentView
-    func setupTextField() {
-        contentView.addSubview(cellTextField)
-        cellTextField.translatesAutoresizingMaskIntoConstraints = false
-        
-        NSLayoutConstraint.activate([
-            cellTextField.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 0.5),
-            cellTextField.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
-            cellTextField.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
-            cellTextField.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -0.5)
-        ])
-    }
-    
-    // Function to add imageView in contentView
-    func setupImageView() {
-        contentView.addSubview(imageandCountView)
-        imageandCountView.addSubview(image)
-        imageandCountView.addSubview(countLabel)
-
-        image.translatesAutoresizingMaskIntoConstraints = false
-        countLabel.translatesAutoresizingMaskIntoConstraints = false
-        imageandCountView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(cellTextView)
```

Comment:
Setup textView doesn't tell me much about what this function does. Just like above comment all functions should aim to only do 1-2 things and be well named and well commented. Much of our Swift logic as I am looking back can be hard to re-look back on and figure out what's going on.

Reply:
I already renamed setupTextView() to setTextViewInTableColumn() and also added comment above function.